crates/rolldown/tests/rolldown/misc/preserve_entry_signature/issue-4880/artifacts.snap

```diff
@@ -0,0 +1,53 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+---
+# Assets
+
+## dynamic.js
+
+```js
+import assert from "node:assert";
+
+//#region dynamic.js
+(async () => {
+	const mod = await import("./foo.js");
+	assert.strictEqual(mod.default, "foo");
+})();
+
+//#endregion
+```
+## foo.js
+
+```js
+//#region foo.js
+var foo_default = "foo";
+
+//#endregion
+export { foo_default as default };
+```
+## foo2.js
+
+```js
+import { foo2_default } from "./foo22.js";
```

Comment:
This is expected, since `preserveEntrySignature: false`, the shared part is defined in chunk `foo22.js`